de.dashup.shared/src/main/java/de/dashup/shared/Enums/Theme.java

```diff
@@ -0,0 +1,40 @@
+package de.dashup.shared.Enums;
+
+public enum Theme {
```

Comment:
Same applies here...

Reply:
As I mentioned in the comment before, I wanted to make the code more modular.